### PR TITLE
2 describe gitlab flow

### LINKED ISSUE #2: describe GitLab Flow

### PR DESCRIPTION
**Pull request by GabrieleMessina**
Added documentation for gitlab flow in docs folder with link in readme.md file

### CODE REVIEW

**Review by afornaia — COMMENTED** [review]
fix root page link

**Comment by GabrieleMessina** [comment]
> fix root page link

@afornaia done :)

File: docs/gitlab_flow.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+## How does GitLab Flow work?
+With GitFlow, developers create a develop branch and make that the default, while GitLab Flow works with the ‘main’ branch right away. GitLab Flow incorporates a pre-production branch to make bug fixes before merging changes back to main before going to production. Teams can add as many pre-production branches as needed — for example, from main to test, from test to acceptance, and from acceptance to production.
+
+## Essentially, teams practice feature branching, while also maintaining a separate production branch. Whenever the ‘main’ branch is ready to be deployed, users merge it into the production branch and release. GitLab Flow is often used with release branches. Teams that require a public API may need to maintain different versions. Using GitLab Flow, teams can make a v1 branch and a v2 branch that can be maintained individually, which can be helpful if the team identifies a bug during code reviews that goes back to v1.
```
Comment (afornaia) [inline docs/gitlab_flow.md]: remove bold case from this block

**Review by afornaia — CHANGES_REQUESTED** [review]
remove bold case from paragraph

**Review by afornaia — APPROVED** [review]
ok

**Pull request merged**